controllers/src/main/scala/cromwell/pipeline/controller/CromwellController.scala

```diff
@@ -0,0 +1,30 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import cromwell.pipeline.datastorage.dto.auth.AccessTokenContent
+import cromwell.pipeline.service.CromwellService
+
+import scala.concurrent.ExecutionContext
+import scala.util.{ Failure, Success }
+
+class CromwellController(cromwellBackendService: CromwellService)(
+  implicit executionContext: ExecutionContext
+) {
+
+  val route: AccessTokenContent => Route = _ => {
+    path("cromwell" / "engine" / "status") {
+      concat(
+        get {
+          onComplete(cromwellBackendService.getEngineStatus()) {
+            case Success(Right(state)) => complete(StatusCodes.OK, state.toString)
```

Comment:
`.toString` is not required, `complete(StatusCodes.OK, state)` should work just fine.